skyvern/forge/sdk/routes/browser_sessions.py

```
@@ -82,7 +82,7 @@ async def get_browser_sessions_all(
     include_in_schema=False,
 )
 async def create_browser_session(
-    browser_session_request: CreateBrowserSessionRequest = CreateBrowserSessionRequest(),
+    browser_session_request: CreateBrowserSessionRequest,
     current_org: Organization = Depends(org_auth_service.get_current_org),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)